lib/model/QubitInformationObject.php

```diff
@@ -1582,6 +1582,37 @@ public function setActorByName($name, $options)
   }
 
   /**
+   * Import data from a <originalsloc> tag in EAD2002
+   *
+   * @param $originalsNode  DOMNode  EAD originalsloc DOM node
+   */
+  public function importOriginalsLocationEadData($originalsNode)
+  {
+    $text = '';
+
+    $pNodes = $originalsNode->getElementsByTagName('p');
```

Comment:
Can probably just leave out the intermediate variables here and stick  $originalsNode->getElementsByTagName('p') directly in the foreach. Same for L1600 & L1604?